src/java/picard/sam/SamToFastq.java

```diff
@@ -90,9 +100,9 @@
             optional = true, mutex = {"FASTQ", "SECOND_END_FASTQ", "UNPAIRED_FASTQ"})
     public boolean OUTPUT_PER_RG;
 
-    @Option(shortName="RGT", doc = "The read group tag (PU or ID) to be used to output a fastq file per read group.")
-    public String RG_TAG = "PU";
-
+    @Option(shortName="RGT", doc = "The read group tag (PU or ID) to be used to output a fastq file per read group.")

```

Comment:
Oh just realized this is technically out of scope since it's an argument doc, not the tool doc itself. Feel free to leave this for a future round.